src/webgl/p5.RendererGL.js

```diff
@@ -970,31 +985,42 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     }
     let pg = this.filterGraphicsLayer;
 
+    // use internal shader for filter constants BLUR, INVERT, etc
+    let filterParameter = undefined;
```

Comment:
lets predefine some defaults for this value. 
```
if(operation === 'BLUR'){
  filterParameter = 4;
} else if(operation == 'POSTERIZE') ...
```
Then later you can check if there is a value in args[1] and set it equal to filterParameter if it exists.